docs/guide/dev/integration/angular/README.md

```diff
@@ -0,0 +1,241 @@
+---
+category: integration
+order: 60
+url: guide/dev_angular
+menu-title: Angular Integration
+meta-title-short: Angular Integration
+---
+<!--
+Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
+For licensing, see LICENSE.md.
+-->
+
+# Angular Integration
+
+<info-box info=""> This feature is provided through the <a href="https://www.npmjs.com/package/ckeditor4-angular"><code>ckeditor4-angular</code> npm package</a>.
+</info-box>
+
+CKEditor offers native Angular integration through `CKEditor` Angular component. It provides deep integration of CKEditor and Angular that lets you use the native features of CKEditor inside Angular component. The `CKEditor` Angular component is compatible with Angular versions 2.0+.
+
+## Basic Usage
+
+In order to create editor instance in Angular, install `ckeditor4-angular` npm package as a dependency of your project:
+
+```bash
+npm install --save ckeditor4-angular
+```
+
+After installing, import `CKEditorModule` to your application:
+
+```typescript
+import CKEditorModule from 'ckeditor4-angular';
+
+@NgModule( {
+	imports: [
+	...
+	CKEditorModule,
+	...
+	],
+	...
+} )
+```
+
+Now you can use `<ckeditor>` tag in component template to run the rich text editor:
+
+```html
+<ckeditor data="<p>Hello, world!</p>"></ckeditor>
+```
+
+The `data` attribute used in the above example is responsible for setting the editor's data.
+
+## Customizing CKEditor Preset or Version
+
+By default `CKEditor` Angular component loads standard preset of the latest CKEditor release from <a href="https://cdn.ckeditor.com/">CDN</a> while creating the first editor. This behavior can be altered by setting the value of `editorUrl` attribute in template to point to desired CKEditor location:
+
+```html
+<ckeditor [editorUrl]="https://your-website.example/ckeditor/ckeditor.js"></ckeditor>
+```
+
+Note that attribute has to be assigned **before first component is initialized**.
+
+Alternatively you can load CKEditor before loading `CKEditor` Angular component. In that case the component will use the already loaded CKEditor:
+
+```html
+<script src="custom-ckeditor.js"></script>
+<script src="main.js"></script>
+```
+
+## Choosing Editor Type
+
+By default `CKEditor` Angular component creates {@link guide/dev/framed/README classic editor}. To create {@link guide/dev/inline/README inline editor} `type` property with value of `inline` must be added:
```

Comment:
From what I know, currently there's no support for classic editor in our angular integration. Is this section relevant to this integration?